packages/ubuntu_wsl_setup/lib/pages/profile_setup/profile_setup_page.dart

```diff
@@ -42,6 +42,19 @@ class _ProfileSetupPageState extends State<ProfileSetupPage> {
 
     final model = Provider.of<ProfileSetupModel>(context, listen: false);
     model.loadProfileSetup();
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      final lang = AppLocalizations.of(context);
+      model.setUsernameValidators(
+        {
+          UsernameValidation.ALREADY_IN_USE: lang.profileSetupUsernameInUse,
+          UsernameValidation.SYSTEM_RESERVED:
+              lang.profileSetupUsernameSystemReserved,
+          UsernameValidation.INVALID_CHARS:
+              lang.profileSetupUsernameInvalidChars,
+          UsernameValidation.TOO_LONG: lang.profileSetupUsernameTooLong,
+        },
+      );
+    });
```

Comment:
enum extensions work great for localizing enum values. for example, https://github.com/canonical/ubuntu-desktop-installer/blob/main/packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/allocate_disk_space_dialogs.dart#L129